.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: Continuous Integration
+
+on:
+    pull_request: ~
+    push: ~
+
+jobs:
+    tests:
+        runs-on: ubuntu-20.04
+        strategy:
+            max-parallel: 1
+            matrix:
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+                    - '8.0'
+                include:
```

Comment:
Travis also included a job running with `minimum-stability: dev`, which got lost in the migration

Reply:
Do you see real value in it? Should we test for some features that vendors still have to release?

Reply:
this is about testing compatibility with the future release of Symfony before it is released. this allows to prepare compat so that we have a chance of being compatible with these version the day they become stable (and users use them)